src/extension.ts

```diff
@@ -38,6 +38,7 @@ export function activate(context: vscode.ExtensionContext) {
                 }
 
                 if (isGhcInstalled && isCabalInstalled) {
+                    await vscode.workspace.getConfiguration().update('editor.tabCompletion', 'onlySnippets', vscode.ConfigurationTarget.Global);
```

Comment:
This would only work on the use of the `install syrup` command and would override all other tab completion for other langs

Reply:
so my original thoughts were that people who use this extension wouldn't have syrup locally downloaded anyways so they would always run this command, i can see the concern of other tab completion so instead i might put a wee comment in read me for user to decide if they wish to have it on or not as most of the snippets work without it just a few i think are useful don't